politeiad/cmd/legacypoliteia/blobs.go

```diff
@@ -0,0 +1,255 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+
+	backend "github.com/decred/politeia/politeiad/backendv2"
+	"github.com/decred/politeia/politeiad/backendv2/tstorebe/store"
+	"github.com/decred/politeia/politeiad/plugins/comments"
+	"github.com/decred/politeia/politeiad/plugins/ticketvote"
+	v1 "github.com/decred/politeia/politeiawww/api/ticketvote/v1"
+	"github.com/decred/politeia/politeiawww/client"
+)
+
+// serverPubkey is the former politeia public key from when it ran the git
+// backend. It is used to verify the auth details and cast vote details
+// signatures.
+const serverPubkey = "a70134196c3cdf3f85f8af6abaa38c15feb7bccf5e6d3db6212358363465e502"
+
+// Ticketvote blobs
+
```

Comment:
This file can be simplified into a single function.

`saveBlob(data interface{}, dataDescriptor string) error`